research/global-config/index.md

```diff
@@ -0,0 +1,361 @@
+---
+title: Global configuration
+authors:
+  - mmassari
+sidebar_position: 1
+---
+
+You can customize the service configuration for a user project in two different ways:
+
+1. There is a configuration on the service side which has to be updated by the service team (upon notification from the service users for opt-in or opt-out from the service). This is how you can enable or disable Zuul for the Fedora CI nowadays: .
+   I will refer to this solution as a **top-down solution**.
+2. There is a configuration, related with the service, on the user's project side. This is how packit works today. I will refer to this solution as a **bottom-up solution**.
+
+You can configure services for individual projects in two ways:
+
+1. Top-Down solution: configuration resides on the service side, managed by the service team. Users notify the team to opt in or out. Example: Zuul for Fedora CI uses this approach ([configuration reference](https://pagure.io/fedora-project-config/blob/master/f/resources/fedora-distgits.yaml)).
+
+2. Bottom-Up solution: configuration lives in the user's project repository. Packit currently implements this approach.
+
+The top-down approach represents a global configuration, while the bottom-up solution can reference global or semi-global configurations. Let's analyze both approaches.
+
+# Top-Down solution
+
+The service side maintains project-specific behavior configurations.
+
+## Advantages
+
+- Users don't need configuration files in their projects; they might not like the idea of having a service related file in their projects.
+- Configuration migrations are straightforward for the service team; it is just a change in a file that belongs to the team.
+- Highest performance due to direct configuration access; no need to load and pre-process other files.
+- Quickest implementation; this approach is the same as for the packit-service configuration file.
```

Comment:
For this reason, we can use this to enable dist-git CI for packages before it is run by default. (And nothing else.)